lib/net/networking_utils.js

```diff
@@ -30,13 +31,25 @@ shaka.net.NetworkingUtils = class {
     const type = shaka.net.NetworkingEngine.RequestType.MANIFEST;
 
     const request = shaka.net.NetworkingEngine.makeRequest([uri], retryParams);
-    request.method = 'HEAD';
 
-    const response = await netEngine.request(type, request).promise;
+    try {
+      request.method = 'HEAD';
+      const response = await netEngine.request(type, request).promise;
+
+      // https://bit.ly/2K9s9kf says this header should always be available,
+      // but just to be safe:
+      mimeType = response.headers['content-type'];
+    } catch (error) {
+      if (error && error.code == shaka.util.Error.Code.HTTP_ERROR) {
+        request.method = 'GET';
+        const response = await netEngine.request(type, request).promise;
+
+        // https://bit.ly/2K9s9kf says this header should always be available,
+        // but just to be safe:
+        mimeType = response.headers['content-type'];
+      }
+    }
 
-    // https://bit.ly/2K9s9kf says this header should always be available,
-    // but just to be safe:
-    mimeType = response.headers['content-type'];
     return mimeType ? mimeType.toLowerCase().split(';').shift() : '';
```

Comment:
I think that comment with the bit.ly link was meant to go with the ternary statement here, since it's what is accounting for `content-type` being undefined.

Reply:
Done!